thesis-de/content-de.tex

```diff
@@ -360,6 +362,128 @@ \subsubsection{Durchführung der Evaluierung}
 \end{outline}
 \nl%
 
+\subsection{Schnittstelle - Datenbereitstellung}
+\label{subsec:schnittstelle_datenbereitstellung}
+Neben der herkömmlichen Methode zur Datenabfrage von Datenbank-Systemen über
+einen sogenannten Datenbanktreiber, existiert noch eine weitere Möglichkeit in
+Form von einer sogenannten ``API''. API bedeuter im ausgeschriebenen
```

Comment:
fixed in https://github.com/DCC-org/documents/pull/68/commits/163c9d035ece91d491ebed273b49ae7bb2028366